core/src/parsing/datetime.rs

```diff
@@ -38,9 +39,118 @@ fn numeric_match(
     }
 }
 
-pub fn parse_date<I>(
-    out: &mut Parsed,
-    out_tz: &mut Option<Tz>,
+pub(crate) struct Fields {
+    era: Option<Era>,
+    year: Option<i32>,
+    iso_year: Option<i32>,
+    month: Option<i32>,
+    week: Option<i32>,
+    weekday: Option<Weekday>,
+    day: Option<i32>,
+    ordinal: Option<i32>,
+
+    hour: Option<i32>,
+    hour12: Option<i32>,
+    meridiem: Option<Meridiem>,
+    minute: Option<i32>,
+    second: Option<i32>,
+    nanosecond: Option<i32>,
+
+    unix_timestamp: Option<i64>,
+    time_zone: Option<TimeZone>,
```

Comment:
If you were using `chrono::format::Parsed` before, you might be able to use `jiff::fmt::strtime::BrokenDownTime`.